source/features/open-all-notifications.tsx

```diff
@@ -27,55 +24,24 @@ function openNotifications({delegateTarget}: delegate.Event): void {
 	});
 
 	// Mark all as read
-	for (const notification of select.all('.unread', container)) {
-		notification.classList.replace('unread', 'read');
+	for (const notification of select.all(unreadNotificationsClass, container)) {
+		notification.classList.replace('notification-unread', 'notification-read');
 	}
 
 	// Remove all now-useless buttons
 	for (const button of select.all(`
 		.rgh-open-notifications-button,
-		.open-repo-notifications,
-		.mark-all-as-read,
-		[href='#mark_as_read_confirm_box']
+		.js-notifications-mark-all,
 	`, container)) {
 		button.remove();
 	}
 }
 
-function addOpenReposButton(): void {
-	for (const repoNotifications of select.all('.boxed-group')) {
-		if (select.exists('.open-repo-notifications', repoNotifications)) {
-			continue;
-		}
-
-		const unreadCount = select.all('.unread', repoNotifications).length;
-		if (unreadCount === 0) {
-			continue;
-		}
-
-		const [, repo] = select<HTMLAnchorElement>('.notifications-repo-link', repoNotifications)!.title.split('/');
-
-		select('.mark-all-as-read', repoNotifications)!.before(
-			<button type="button" className="open-repo-notifications tooltipped tooltipped-w rgh-open-notifications-button" aria-label={`Open all unread \`${repo}\` notifications in tabs`}>
-				<LinkExternalIcon/>
-			</button>
-		);
-	}
-}
-
 function addOpenAllButton(): void {
 	if (!select.exists('.rgh-open-notifications-button')) {
-		// Move out the extra node that messes with .BtnGroup-item:last-child
-		document.body.append(select('#mark_as_read_confirm_box') ?? '');
-
-		// Create an open button and add it into a button group
-		const button = <button className="btn btn-sm rgh-open-notifications-button" type="button">Open all unread in tabs</button>;
-		select('.tabnav .float-right')!.prepend(button);
-
-		// There is no sibling on `/<org>/<repo>/notifications` page
-		if (button.nextElementSibling) {
-			groupButtons([button, button.nextElementSibling]);
-		}
+		// Create an open button and add it to the DOM
+		const button = <button className="btn btn-sm ml-2 rgh-open-notifications-button" type="button">Open all unread in tabs</button>;
+		select('.notifications-list .Box-header')!.append(button);
```

Comment:
This fails when grouping by repository:

<img width="461" alt="Screen Shot 2020-05-01 at 16 32 01" src="https://user-images.githubusercontent.com/1402241/80813007-753f6e00-8bc9-11ea-9ab5-73030066ba65.png">

Reply:
Also I'd keep the icon there and I'd skip the `button` variable. Example:

```ts
const button = ;
select('.notifications-list .Box-header')!.append(
	<button className="btn btn-sm ml-2 rgh-open-notifications-button" type="button">
		<LinkExternalIcon/> Open all unread in tabs
	</button>
);
```

Reply:
Location updated see comments 😉

Reply:
Neither one of my suggestion was taken. Please resolve

Reply:
Grouping by repository works now